packages/tasks/src/model-libraries.ts

```diff
@@ -933,6 +933,14 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: true,
 		countDownloads: `path:"pytorch_model.bin" OR path:"model.safetensors"`,
 	},
+	torchgeo: {
+		prettyLabel: "TorchGeo",
+		repoName: "torchgeo",
+		repoUrl: "https://github.com/microsoft/torchgeo",
+		docsUrl: "https://torchgeo.readthedocs.io/",
+		filter: false,
+		countDownloads: `path_extension:"pt" OR path_extension:"pth"`,
```

Comment:
I guess https://huggingface.co/torchgeo/ftw is the kind of repo that should be served with torchgeo? In that case, we strongly recommend to follow the 1 model weights == 1 repo. Doing this ensures you can properly document differences between 2 models, users can better find them on the Hub, we can count downloads separately, community tab is made "per model", model lineage can be better tracked, etc. Usually when multiple repos are created to store multiple variants of a model, the filename is renamed to be the same across all repos - making it more consistent. If that's the case, then it's better to update `countDownloads` with that filename. 

The above is optional though. We strongly recommend it but don't enforce it.

Reply:
@adamjstewart thoughts on these changes to how we store the model weights?

Reply:
+1 on one model weight == 1 repo - it also increases the visibility of individual models

Reply:
I'm not opposed to this. I've actually asked this same question before but didn't get a response: https://discuss.huggingface.co/t/how-to-organize-hundreds-of-pre-trained-models/42682. Glad to know there are pros of separate repos per model.

@isaaccorley if you want to work on porting these, just remember to keep the existing model repos intact so that older TorchGeo releases aren't broken. This would be good to do at the same time as your .pt2 conversion.

> Usually when multiple repos are created to store multiple variants of a model, the filename is renamed to be the same across all repos - making it more consistent.

This doesn't seem to be compatible with the `check_hash` feature of https://docs.pytorch.org/docs/stable/hub.html#torch.hub.load_state_dict_from_url

Reply:
For backwards compatibility we will keep the old repos but I will work on porting these to new 1 model 1 repo approach now.

Reply:
I just replied to the forum post for future users landing on it (with 2 years delay...^^). Sorry about that! :grimacing: 

> For backwards compatibility we will keep the old repos but I will work on porting these to new 1 model 1 repo approach now.

Sound good!

>> Usually when multiple repos are created to store multiple variants of a model, the filename is renamed to be the same across all repos - making it more consistent.

> This doesn't seem to be compatible with the check_hash feature of https://docs.pytorch.org/docs/stable/hub.html#torch.hub.load_state_dict_from_url

I'm perfectly fine keeping `countDownloads: `path_extension:"pt" OR path_extension:"pth"`,` then!